src/OpenTok/Util/Client.php

```diff
@@ -64,6 +67,14 @@ public function configure($apiKey, $apiSecret, $apiUrl, $options = array())
         $this->apiKey = $apiKey;
         $this->apiSecret = $apiSecret;
 
+        if (array_key_exists('application_id', $options) || array_key_exists('private_key_path', $options)) {
```

Comment:
We should probably just take an extra flag in this `$options` that signifies if we are using Vonage or Opentok, so that we can flip between the correct type of auth (JWT or T1, and eventually the new OT JWTs) instead of checking for app ID